ICAFIX/hcp_fix_multi_run

```diff
@@ -352,6 +359,17 @@ then
 	log_Err_Abort "--fallback-threshold is not allowed when --vol-wisharts=1 is used"
 fi
 
+case "$ICAmethod" in
+	(MELODIC|ICASSO)
+		;;
+	(*)
+		log_Err_Abort "--ica-method=${ICAmethod} must be MELODIC or ICASSO"
+		;;
+esac
+if [[ ${FSL_FIX_MATLAB_MODE} == 0 ]] && [[ ${ICAmethod} == ICASSO ]];then
```

Comment:
Again, I don't think we want this type of check in the committed version.  We can compile it as a final step before merging.